radish34/contracts/package.json

```diff
@@ -5,19 +5,19 @@
   "main": "src/utils.js",
   "scripts": {
     "test": "jest --runInBand --forceExit --detectOpenHandles",
-    "build": "sol-compiler"
+    "build": "truffle compile"
```

Comment:
I'd like to give the cheeky suggestion that `etherlime` will do the same thing and not add web3.js to our dependencies ;)

Reply:
@Perseverance I'm not very familiar with `etherlime`'s capabilities. Are you suggesting to use it to compile the contracts rather than using `truffle`?

Reply:
should we consider a switch or an option to be added to use etherlime instead? Does etherlime use truffle as well? @Perseverance